src/test/java/io/jenkins/plugins/appcenter/task/internal/UploadAppToResourceTaskTest.java

```diff
@@ -95,6 +96,29 @@ public void should_ReturnDebugSymbolUploadId_When_DebugSymbolsAreFound() throws
             .isEqualTo(request);
     }
 
+    @Test
+    @Ignore("Unable to test chunked uploads due to inability to mock BlobStorage server using MockWebserver.")
```

Comment:
This would have been the test I wanted to write if it were possible to mock out Azure Blob storage in the same way that we mock our the AppCenter server using MockWebserver for our other unit tests.

But this will have to wait....